brainstormforce/astra
# AST-1542 - There is no any gap between post edit title & first content block in block editor

### Description
- There is markup change in WP-5.9 - https://share.bsf.io/9ZuDdE8D
- IMO, let go with WordPress default styling & offerings that's why removed the old CSS for boxed-container style
- That's why removed negative margin old CSS
- Checked on existing setup as well, its not creating major difference, looks good on contrary

### Screenshots
- https://share.bsf.io/llu6NDEx

### Types of changes
- Bug fix 

### How has this been tested?
- Check with boxed/content-boxed layout
- Add new post/page in WP-5.9 & here will get the issue
- Checkout to this branch
- Empyt cache & hard reload it, that should fix the issue

### Checklist:
